azure-pipelines.yml

```diff
@@ -1,4 +1,37 @@
 jobs:
+- job: DPCPP
+  pool:
+    vmImage: 'ubuntu-18.04'
+  steps:
+  - script: |
+      wget https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2023.PUB
+      sudo apt-key add GPG-PUB-KEY-INTEL-SW-PRODUCTS-2023.PUB
+      rm GPG-PUB-KEY-INTEL-SW-PRODUCTS-2023.PUB
+      echo "deb https://apt.repos.intel.com/oneapi all main" | sudo tee /etc/apt/sources.list.d/oneAPI.list
+      sudo add-apt-repository "deb https://apt.repos.intel.com/oneapi all main"
+      sudo apt-get update
+      sudo apt-get install              \
+          intel-oneapi-common-vars      \
+          intel-oneapi-common-licensing \
+          intel-oneapi-tbb-devel        \
+          intel-oneapi-daal-devel       \
+          intel-oneapi-dpcpp-compiler   \
+          intel-oneapi-dev-utilities    \
+          intel-oneapi-libdpstd-devel   \
+          cmake
+      sudo bash -c 'echo libintelocl.so > /etc/OpenCL/vendors/intel-cpu.icd'
+      sudo mv /opt/intel/inteloneapi/compiler/latest/linux/lib/oclfpga /opt/intel/inteloneapi/compiler/latest/linux/lib/oclfpga_
+    displayName: 'apt-get'
+  - script: conda create -q -y -n CB python=3.7 conda-build conda-verify
+    displayName: Create Anaconda environment
+  - script: |
+      export DPCPP_VAR=/opt/intel/inteloneapi/compiler/latest/env/vars.sh
+      export DAALROOT=/opt/intel/inteloneapi/daal/latest
+      . /usr/share/miniconda/etc/profile.d/conda.sh
+      conda activate CB
+      conda build --override-channels -c conda-forge -c intel conda-recipe
```

Comment:
you want to prefer the conda-forge channel?

Reply:
yes, it is our considered decision